src/graph/executor/algo/SingleShortestPath.cpp

```diff
@@ -264,6 +264,9 @@ folly::Future<bool> SingleShortestPath::buildEvenPath(size_t rowNum,
       return false;
     }
     for (auto& meetVertex : vertices) {
+      if (!meetVertex.isVertex()) {
```

Comment:
Why this isn't vertex?

Reply:
vertices may contain EMPTY values ​​in exceptional cases